src/schema/v2/artwork/index.ts

```diff
@@ -258,6 +258,13 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
   name: "Artwork",
   interfaces: [NodeInterface, Searchable, Sellable],
   fields: () => {
+    // Dynamically require to avoid circular dependency
+    const { OrderType } = require("../order/types/OrderType")
+    const ArtworkOrdersConnectionType = connectionWithCursorInfo({
```

Comment:
Should this type be defined in the other file and required here, versus defining the connection type here?

Reply:
Good point.